lib/modules/datasource/git-tags/index.spec.ts

```diff
@@ -14,49 +15,105 @@ const datasource = GitTagsDatasource.id;
 const datasourceInstance = new GitTagsDatasource();
 
 describe('modules/datasource/git-tags/index', () => {
+  let gitMock: any;
+
+  beforeEach(() => {
+    // clear host rules
+    clear();
+
+    // clear environment variables
+    process.env = {};
+
+    // reset git mock
+    gitMock = {
+      env: jest.fn(),
+      listRemote: jest.fn(),
+    };
+
+    simpleGitFactoryMock.mockReturnValue(gitMock);
+    gitMock.env.mockImplementation(() => gitMock as unknown as SimpleGit);
+  });
+
   describe('getReleases', () => {
     it('returns nil if response is wrong', async () => {
-      simpleGit.mockReturnValue({
-        listRemote() {
-          return Promise.resolve('') as Response<string>;
-        },
-      });
+      gitMock.listRemote.mockResolvedValue('');
+
       const versions = await getPkgReleases({ datasource, packageName });
       expect(versions).toBeNull();
     });
 
     it('returns nil if remote call throws exception', async () => {
-      simpleGit.mockReturnValue({
-        listRemote() {
-          throw new Error();
-        },
-      });
+      gitMock.listRemote.mockRejectedValue(new Error());
+
       const versions = await getPkgReleases({ datasource, packageName });
       expect(versions).toBeNull();
     });
 
     it('returns versions filtered from tags', async () => {
-      simpleGit.mockReturnValue({
-        listRemote() {
-          return Promise.resolve(lsRemote1) as Response<string>;
-        },
+      gitMock.listRemote.mockResolvedValue(lsRemote1);
+
+      const versions = await getPkgReleases({
+        datasource,
+        packageName,
+      });
+      expect(versions).toMatchSnapshot();
```

Comment:
Do not add additional `toMatchSnapshot` rather use `toEqual` or `toMatchObject`.

Reply:
done in 544e0e7f1e773f0d0f50bf8e7709c906e9ea3680; I moved the tests to only check for the digest, which is enough for what I want to ensure and that we can make inline.